interface-definitions/include/accel-ppp/client-ip-pool.xml.i

```diff
@@ -0,0 +1,46 @@
+<!-- include start from accel-ppp/client-ip-pool.xml.i -->
+<tagNode name="client-ip-pool">
+  <properties>
+    <help>Client IP Pool</help>
+    <valueHelp>
+      <format>txt</format>
+      <description>Name of IP pool</description>
+    </valueHelp>
+    <constraint>
+      <regex>[-_a-zA-Z0-9.]+</regex>
```

Comment:
I can't remember if accel-ppp has constraints on pool names, but this regex feels too lax, e.g., it allows `...` or `-._.-` as valid pool names.

Reply:
I have just copied it from